### DISCUSSION TITLE
Next target release

### DISCUSSION THREAD

**Discussion by phirrip** [GSI-CS-CO/chart-fx#509] [General]
I’m really keen to get the series visibility feature out to my users. Do you have a target date for the next release?

Similarly - do you have any plans or expectations for release frequency going forward?

**Comment by RalphSteinhagen** [comment]
@phirrip the next micro-release is the next item on our todo list.

We are preparing a minor release with some fundamental internal but beneficial changes that will also involve a change of the authority from `de.gsi.*` to `io.fair-acc.*`.

We have been a bit delayed with the upgrades due to some other parallel projects but aim normally at a minor/micro release at least twice per year or whenever there are enough commits/PRs and/or qualified bug fixes/security updates.

**Reply by phirrip** [reply]
Great to hear!
